pkg/installer/build.yml

```diff
@@ -4,14 +4,14 @@
 # SPDX-License-Identifier: Apache-2.0
 org: lfedge
 image: eve-installer
+network: no
```

Comment:
Isn't the syntax "net: no"?

Reply:
Checkout pkg/alpine, pkg/bpftrace, pkg/fscrypt, etc. I believe net syntax is also valid if we have config, like pkg/debug.

Reply:
So, I could have the yml like: 

```
org: lfedge
image: eve-installer
config:
  net: no
  binds:
    - /dev:/dev
    - /lib/modules:/lib/modules
    - /etc/resolv.conf:/etc/resolv.conf
    - /run:/run
    - /:/root
  capabilities:
    - all
  pid: host
  rootfsPropagation: shared
  devices:
    - path: all
      type: a
```

Reply:
Seems like you're doing it right: https://github.com/linuxkit/linuxkit/blob/master/src/cmd/linuxkit/pkglib/pkglib.go#L27 ;-)

Reply:
So it was the old "net: host" that was incorrect?
Or it does prefix matching?

Anyhow, no need to change. Just trying to understand.

Reply:
It is 2 different things (and confusing). At the top-level of `build.yml` is `network: no` (or similar), which is "does this container have access to the network during build phase". It is equivalent to `docker build --network no` (or `yes`, etc.).

Lower down, under `config`, it is the config information that is saved along with the container image, and is used as a basis by containerd to create the config that runc uses when launching the container. That is `net: host`, etc.

Container build-time vs run-time.